apps/DesktopStreamer/main.cpp

```diff
@@ -40,14 +40,16 @@
 
 #include "MainWindow.h"
 
+#include <deflect/utils.h>
 #include <deflect/version.h>
 
 #include <QCommandLineParser>
 
 int main(int argc, char* argv[])
 {
-    QApplication app(argc, argv);
+    deflect::blockFalseWarnings();
```

Comment:
Sorry, the default QtMessageHandler returned is NOT nullptr so it's impossible to check whether there is a handler already registered or not. Each application has to do it explicitly.